slices/python3.yaml

```diff
@@ -0,0 +1,29 @@
+package: python3
+
+essential:
+  - python3_copyright
+
+slices:
+  core:
+    essential:
+      - python3.12_core
+    contents:
+      /usr/bin/python3: {symlink: /usr/bin/python3.12}
```

Comment:
The latest change artificially creates the python3 symlink, but I don't think that's the right way to do it. The symlink is created by python3-minimal which is a dependency that should be included in this PR.